tests/commands/test_images.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+
+from click.testing import CliRunner
+from tutor.commands.images import *
+from tests.test import CONTEXT
+
+
+class ImagesTests(unittest.TestCase):
+    def test_images_help(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(images_command, ["--help"])
+        self.assertEqual(0, result.exit_code)
+        self.assertEqual(None, result.exception)
+
+    def test_images_pull_image(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(images_command, ["pull"], obj=CONTEXT)
+        self.assertEqual(0, result.exit_code)
+        self.assertEqual(None, result.exception)
+
+    def test_images_pull_plugin(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(images_command, ["pull", "plugin"], obj=CONTEXT)
```

Comment:
`tutor pull plugin` is not an actual command. In this context, `plugin` will be interpreted as an image name, which will not exist. Right now this is not triggering an error, but it will once this other PR is merged: https://github.com/overhangio/tutor/pull/535
So I suggest that we either remove this test or mock the `images.pull` function.

Reply:
I end up mocking the `images.pull` function.